Gruntfile.js

```diff
@@ -42,10 +50,52 @@ module.exports = function(grunt) {
                 authors: 'My App Inc.',
                 exe: 'myapp.exe'
             }
-        }
+        },
+        electron: {
+            osxBuild: {
+                options: {
+                    name: 'azuretools',
+                    dir: 'electronbuildcache',
+                    out: 'builds',
+                    version: '0.36.7',
+                    overwrite: true,
+                    platform: 'darwin',
+                    arch: 'x64'
+                }
+            }
+        },
+        appdmg: {
+            options: {
+                title: 'Azure Tools',
+                icon: './public/icon/ase.icns',
+                background: './public/osx/background.tiff',
+                'icon-size': 156,
+                contents: [{
+                    x: 470,
+                    y: 430,
+                    type: 'link',
+                    path: '/Applications'
+                }, {
+                    x: 120,
+                    y: 430,
+                    type: 'file',
+                    path: './builds/azuretools-darwin-x64/azuretools.app'
+                }]
+            },
+            target: {
+                dest: './builds/Azure Tools-darwin-x64/azuretools.dmg'
+            }
+        },
+        copy: {
+            app: {
+                expand: true,
+                src: ['main.js', 'package.json', 'app/**', 'node_modules/**', 'libs/**'],
```

Comment:
we should include only necessary node_modules, or better to exclude all unnecessary e.g grunt-